.github/workflows/tests.yml

```diff
@@ -160,12 +160,6 @@ jobs:
         run: echo "CPU_MODEL=$(./bin/print_cpu.sh)" >> $GITHUB_ENV
       - name: "Print CPU model"
         run: echo "${{ env.CPU_MODEL}}"
-      # --- Cache ---
```

Comment:
Unfortunately, WAMR machine code generated with one version of the system might not be compatible with other versions. Consequently, tests for which machine code was cached would fail for no aparent reason.

I know this makes test runs even slower that they currently are. I suggest we reintroduce the cache when the WAMR/SGX build and code is consolidated.